less/mixins.less

```diff
@@ -1 +1,14 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.border-bottom() {
+    border-bottom: 2px @navigation-border dashed;
```

Comment:
seeing that there is only 1 style here.  It might be better to set a variable instead of a mixin here